LISAv2-Framework.psm1

```diff
@@ -186,7 +187,7 @@ function Start-LISAv2 {
 			Write-LogInfo "Test $global:testId finished"
 
 			# Output text summary
-			$plainTextSummary = $testController.TestSummary.GetPlainTextSummary($OsVHD, $ARMImageName, $OverrideVMSize)
```

Comment:
Did we remove [string] $ARMImageName ? It will be unused variable.

Reply:
No, it's still a valid Run-LISAv2 parameter. which could not be removed, due to backward compatible.